### PR TITLE
Bugfix: ColorPicker color changing adds multiple entries in undo stack

### PR DESCRIPTION
**Pull request by KhalfounMehdi**
The bug is described in the following issue : #3890

### CODE REVIEW

**Comment by thegreatercurve** [comment]
@KhalfounMehdi This generally looks fine! Would you mind just fixing the few lint errors?

**Comment by KhalfounMehdi** [comment]
@thegreatercurve  actually i need to fix an bug that I noticed in my fix : 
- after I call the editor.update , the merge action returned is `HISTORY_MERGE` and it should be `HISTORY_PUSH`
I'll finish with the bug , I will fix the lining errors nad push

File: packages/lexical-playground/src/ui/ColorPicker.tsx [hunk 1]
```diff
@@ -99,6 +101,11 @@ export default function ColorPicker({
     setSelfColor(newColor);
     setInputColor(newColor.hex);
   };
+  const HandleDropDownClose = React.useCallback(() => {
```
Comment (thegreatercurve) [inline packages/lexical-playground/src/ui/ColorPicker.tsx:104]:
```suggestion
  const HandleDropDownClose = useCallback(() => {
```

File: packages/lexical-playground/src/ui/DropDown.tsx [hunk 2]
```diff
@@ -199,7 +201,22 @@ export default function DropDown({
       };
     }
   }, [dropDownRef, buttonRef, showDropDown, stopCloseOnClickSelf]);
+  function usePrevious(value: any) {
```
Comment (fantactuka) [inline packages/lexical-playground/src/ui/DropDown.tsx:204]: Unless we want this to be common util, you can just use ref without hook

**Review by zurfyx — CHANGES_REQUESTED** [review]
Can we please revise the behavior of this PR and consider replicating the mouse up behavior from Figma (suggested in https://github.com/facebook/lexical/issues/3890)? The ColorPicker doesn't seem to be working correctly and also breaks the same time you open the picker

https://user-images.githubusercontent.com/193447/233785218-040de80e-ea65-4aa3-880b-c04ab21f7509.mov

**Comment by thegreatercurve** [comment]
Closing due to PR becoming stale

**Pull request closed without merging**